lib/OpenQA/WebAPI/Controller/API/V1/Iso.pm

```diff
@@ -115,7 +122,7 @@ sub _sort_dep {
 
             my $c = 0;    # number of parens that must go to @out before this job
             foreach my $a (@after) {
-                $c += $count{$a} if defined $count{$a};
+                $c += $count{join(':', @$a)} if defined $count{join(':', @$a)};
```

Comment:
I would join it only once and assign it to a variable with a meaningful name. Also the loop variable hasn't a good name.

Reply:
Fixed